services/job_runner.go

```diff
@@ -259,7 +261,7 @@ func executeRunAtBlock(
 		return jr, wrapExecuteRunAtBlockError(jr, errors.New("No unfinished tasks to run"))
 	}
 	offset := len(jr.TaskRuns) - len(unfinished)
-	prevResult, err := unfinished[0].Result.Merge(jr.Overrides)
+	prevResult, err := jr.Overrides.Merge(unfinished[0].Result)
```

Comment:
Interesting. Does this fix #160188160?

https://www.pivotaltracker.com/story/show/160188160

Reply:
Yup!

Reply:
I have removed this fix, as it is a little bit more complicated and will be revisited in part 2 of this feature.